Sources/MapboxCoreNavigation/Directions+RoutingProvider.swift

```diff
@@ -8,30 +8,30 @@ extension URLSessionDataTask: NavigationProviderRequest {
 }
 
 extension Directions: RoutingProvider {
-    
+
     @discardableResult public func calculateRoutes(options: RouteOptions, completionHandler: @escaping RouteCompletionHandler) -> NavigationProviderRequest? {
-        return calculate(options, completionHandler: completionHandler)
+        return calculateWithCache(options, completionHandler: completionHandler)
```

Comment:
@Udumft, was it intentional that this method calls the more generic calculation method as of #3261, versus what MapboxRoutingProvider does?

Reply:
Yes, I intentionally did not use offline capabilities here, to maintain usual Directions behavior. This protocol conformance just wraps Directions features, but since Directions on its own does only online calculations - I didn't use `calculateWithCache`.